src/table/table-row.jsx

```diff
@@ -47,7 +111,6 @@ const TableRow = React.createClass({
   getDefaultProps() {
     return {
       displayBorder: true,
-      displayRowCheckbox: true,
```

Comment:
@oliviertassinari It was not declared in PropType, so it was giving an error by the parser. It was either delete or add it to PropType.